tox.ini

```diff
@@ -4,29 +4,20 @@
 # and then run "tox" from this directory.
 
 [tox]
-envlist = docs, py{27,34}pep8, py{26,27,33,34,py}, py{26,27,33,34,py}-defusedxml, py{27,34}-coverage
+envlist = testenv, py36-coverage, py36pep8
```

Comment:
We don't care about Python 2.x or anything older than 3.6 (what we run on production). Some of these (like "docs") aren't even defined below.

Reply:
Just checking: `nosetests --with-doctest []` (L11) doesn't need `docs`?

And `defusedxml` was one of the ones that was unused as well?

Reply:
docs & defusedxml were never defined as far as I can tell. "--with-doctest" is called but I don't see any doc tests.